src/app/modules/dashboard/container/auth-dashboard/auth-dashboard.component.spec.ts

```
@@ -0,0 +1,25 @@
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ AuthDashboardComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(AuthDashboardComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)